src/components/Breadcrumb/Breadcrumb.less

```diff
@@ -7,45 +7,165 @@
 
-    &:hover {
-      color: @ms-color-black;
+  .ms-Breadcrumb-listItem {
+    list-style-type: none;
+    vertical-align: top;
+    margin: 0;
+    padding: 0;
+    display: inline-block;
+
+    &:last-of-type {
+      .ms-Icon--chevronRight {
+        display: none;
+      }
     }
+  }
+}
 
-    &:active {
-      color: @ms-color-themePrimary;
+.ms-Breadcrumb-overflow {
+  display: none;
+  position: relative;
+  margin-right: -4px;
+
+  .ms-Breadcrumb-overflowButton {
+    font-size: @ms-font-size-s;
+    display: inline-block;
+    color:@ms-color-themePrimary;
+    margin-right: -4px;
+    padding: 12px 8px 3px 8px;
+    cursor: pointer;
+  }
+}
+
+.ms-Breadcrumb-overflowMenu {
+  display: none;
+  position: absolute;
+
+  &.is-open {
+    display: block;
+    top: 36px;
+    left: 0;
+    .drop-shadow;
+    background-color: @ms-color-white;
+    border: 1px solid @ms-color-neutralTertiaryAlt;
+    z-index: (@ms-zIndex-ContextualMenu + @ms-zIndex-middle);
+  }
+
+  &:before {
+    position: absolute;
+    .drop-shadow;
+    top: -6px;
+    left: 6px;
+    content: ' ';
+    width: 16px;
+    height: 16px;
+    transform: rotate(45deg);
+    background-color: white;
+  }
+
+  .ms-ContextualMenu {
+    border: none;
+    box-shadow: none;
+    position: relative;
+    width: 190px;
+
+    &.is-open {
+      margin-bottom: 0;
     }
   }
+}
+
+.ms-Breadcrumb-itemLink,
+.ms-Breadcrumb-overflowButton {
+  text-decoration: none;
+  outline: transparent;
 
-  // Right-pointing chevron
-  // TODO: Localization, point arrow to left.
-  .ms-Breadcrumb-chevron {
-    padding: 0 7px 0 5px;
+  &:hover {
+    background-color: @ms-color-neutralLighter;
   }
 
-  // Only show the big title on large and extra large displays.
-  @media (min-width: @ms-screen-lg-min) {
-    .ms-Breadcrumb-currentLarge {
-      display: inline;
-      font-family: @ms-font-family-regular;
-      font-size: @ms-font-size-xl;
-      color: @ms-color-themePrimary;
-      line-height: 42px;
-      margin-right: 20px;
+  &:focus {
+    outline: @ms-color-neutralSecondaryAlt solid 1px;
+    color: @ms-color-black;
+  }
+
+  &:active {
+    outline: transparent;
+    background-color: @ms-color-neutralTertiaryAlt;
+  }
+}
+
+.ms-Breadcrumb-itemLink {
+  .ms-font-xl();
+  display: inline-block;
+  padding: 0 4px;
+  max-width: 160px;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  overflow: hidden;
+}
+
+
+@media screen and (max-width: 640px) {
+  .ms-Breadcrumb {
+    margin: 10px 0;
+
+    .ms-Icon--chevronRight {
+      font-size: @ms-font-size-m;
+      margin-top: 7px;
+    }
+  }
+
+  .ms-Breadcrumb-itemLink {
+    font-size: @ms-font-size-l;
+  }
+
+  .ms-Breadcrumb-overflow {
+    .ms-Breadcrumb-overflowButton {
+      padding-top: 8px;
+      padding-bottom: 3px;
+    }
+  }
+}
+
+@media screen and (max-width: 480px) {
+  .ms-Breadcrumb-itemLink {
+    font-size: @ms-font-size-m;
+    max-width: 116px;
+  }
+
+  .ms-Breadcrumb {
+    .ms-Icon--chevronRight {
```

Comment:
Maybe we should put a custom class like .ms-Breadcrumb-chevron or something just in case someone in the future wanted to use the chevron class inside the breadcrumb. Thoughts?